app/src/main/java/app/revanced/integrations/patches/LithoFilterPatch.java

```diff
@@ -206,6 +206,11 @@ public GeneralBytecodeAdsPatch() {
         var suggestions = new BlockRule(SettingsEnum.ADREMOVER_SUGGESTIONS_REMOVAL, "horizontal_video_shelf");
         var latestPosts = new BlockRule(SettingsEnum.ADREMOVER_HIDE_LATEST_POSTS, "post_shelf");
         var channelGuidelines = new BlockRule(SettingsEnum.ADREMOVER_HIDE_CHANNEL_GUIDELINES, "channel_guidelines_entry_banner");
+        var previewComment = new BlockRule(
+                SettingsEnum.HIDE_PREVIEW_COMMENT,
+                "carousel_item",
+                "comments_entry_point_teaser",
+                "comments_entry_point_simplebox");
```

Comment:
```suggestion
                "comments_entry_point_simplebox"
        );
```

Please be more consistent in formatting.